src/java/org/tensorics/core/booleans/operations/LogicalOperationsRepository.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2016 European Organisation for Nuclear Research (CERN), All Rights Reserved.
+ */
+
+package org.tensorics.core.booleans.operations;
+
+import org.tensorics.core.math.operations.BinaryOperation;
+
+/**
+ * Repository for logical operations <lu><li>AND</li><li>NAND</li><li>OR</li><li>XOR</li></lu>
+ * 
+ * @author agorzaws
+ */
+public final class LogicalOperationsRepository {
+
+    private LogicalOperationsRepository() {
+        /* only static methods */
+    }
+
+    public static BinaryOperation<Boolean> and() {
```

Comment:
In principle these are immutables and could be public static final fields (constants). But perhaps methods are more consistent with the other repositories (as defining functional constants was awkward before java8), so fine for me.